Resolve 4# Added new footer feature

abalanag opened the pull request:
Added a new feature with the following modifications:
* Created the project footer on the footer.html file
* Created the link to the Bootstrap styles on the common html file

abalanag committed:
4# Added new footer feature

Added a new feature with the following modifications:
* Created the project footer on the footer.html file
* Created the link to the Bootstrap styles on the common html file

danutchindris commented on src/main/resources/templates/common.html:
```diff
@@ -0,0 +1,17 @@
+<body>
+
+</body>
+</html>
```
Please always add an empty line to each file you commit to the Git repository.

abalanag replied:
Empty line added

danutchindris commented on src/main/resources/templates/footer.html:
```diff
@@ -0,0 +1,41 @@
+    </div>
+</footer>
+</body>
+</html>
```
Same as above.

abalanag replied:
Empty line added

danutchindris commented on src/main/resources/templates/footer.html:
```diff
@@ -0,0 +1,41 @@
+
+</head>
+<body>
+<footer class="text-center text-white" style="background-color: rgba(0, 0, 0, 0.2);" th:fragment="footer">
```
Adding a `style` attribute is discouraged. This should be used only as a last resort.
We should use cascading style sheets whenever possible.

abalanag replied:
Created style.css file in which was added the background color.

danutchindris commented on src/main/resources/templates/footer.html:
```diff
@@ -0,0 +1,41 @@
+
+
+    <!-- Copyright -->
+    <div class="text-center text-dark p-3"> © 2020 Copyright:
```
Use `&copy;` instead.

danutchindris replied:
It's 2021 already 😁

abalanag replied:
Traveled back to the year 2021, line updated.

danutchindris commented on src/main/resources/templates/footer.html:
```diff
@@ -0,0 +1,41 @@
+
+    <!-- Copyright -->
+    <div class="text-center text-dark p-3"> © 2020 Copyright:
+        <a class="text-dark" href="//localhost">Alin AirLines</a>
```
Write a proper URL for this anchor element, using the Thymeleaf specific attribute.

abalanag replied:
I hope this will make the trick 😄

danutchindris commented on src/main/resources/templates/footer.html:
```diff
@@ -0,0 +1,41 @@
+        <section class="mb-4">
+            <!-- Facebook -->
+            <a href="https://google.com" type="button" class="btn btn-secondary">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
```
Is there no other solution than writing these `svg` elements inside the markup?
Maybe you can have actual SVG files as static resources and reference them in the markup.

abalanag replied:
Created SVG files for all the images and updated lines.

danutchindris reviewed (changes requested):
Please see the review comments and perform the necessary changes.
Thanks!

abalanag committed:
#4 Updated footer

Updated footer taking into consideration all the comments from the pull request

danutchindris reviewed (approved):
OK!
